edit.py

```diff
@@ -60,6 +60,9 @@ def read_file(path, buf):
         with open(PATH) as f:
             text = f.read()
 
+        if text[-1] == "\n":
+            text = text[:-1]  # remove trailing newline if exists.
+
```

Comment:
```suggestion
            text = f.read().strip()
```

hrm should we just `.strip()` instead? probably fine to not allow any leading space either?

Reply:
I don't know... If leading space is included, wouldn't that be intentional?
Same if multiple trailing newlines are included. Strip would remove them all, while [:-1] only removes one, allowing trailing newlines to be included intentionally if desired.
Both work though I guess...

Reply:
i think it should be ok 🙃